### ISSUE TITLE
Issue with "project delete"

### ISSUE THREAD

**Issue by mreferre** [aws/amazon-ecs-cli-v2#666]
Hi there. 

I am trying to delete a project after deleting the app deployments and the environments. 

I get this (I am also showing the content of `config` and `credentials` just in case): 
```
sh-4.2# ecs project delete                                                                                                                                       

? Are you sure you want to delete project demo? Yes
✘ Error deleting project resources.
Error: delete project resources: DeleteStackInstances for stackset demo-infrastructure, accounts [], and regions []: ValidationError: Regions list cannot be null or empty
        status code: 400, request id: 556e41eb-4789-4c7b-8eb0-3a16fb5aa875
sh-4.2# cat /root/.aws/config 
[default]
region = us-west-2
sh-4.2# cat /root/.aws/credentials 
[default]
region = us-west-2
sh-4.2# 
```
I don't have any env or app right now: 
```
sh-4.2# ecs app ls 
Name                Type
----                ----
sh-4.2# 
sh-4.2# ecs env ls
sh-4.2# 
```

Needless to say I was able to create projects, envs, apps, deployments with the current CLI setup.

Is this supposed to work or am I missing something in the syntax?

**Comment by efekarakus** [comment]
Hello!

Are you using the latest released binary (v0.0.5), or did you build it from scratch?

We recently fixed a bug in `project delete`: https://github.com/aws/amazon-ecs-cli-v2/commit/c32c916b0d81c396c8cda4561f7884a2a87019e2 which I think will fix this issue but we haven't released the binary yet.

Can you try building the latest binary by running `make build`? you can find the binary under `bin/local/ecs-preview`. Alternatively with the Dockerfile:
```
git clone https://github.com/aws/amazon-ecs-cli-v2
cd amazon-ecs-cli-v2
docker build -t aws/amazon-ecs-cli-v2 . &&\
docker create -ti --name amazon-ecs-cli-v2-builder aws/amazon-ecs-cli-v2 &&\
docker cp amazon-ecs-cli-v2-builder:/aws-amazon-ecs-cli-v2/bin/local/ .
```

**Comment by kohidave** [comment]
I can confirm that that change will fix that issue :)
